backend/src/agents/tools/auth_tools.py

```
@@ -0,0 +1,537 @@
+        jwt_pattern = re.compile(r"eyJ[A-Za-z0-9_\-]+\.[A-Za-z0-9_\-]+\.[A-Za-z0-9_\-]+")
+
+        found_tokens = []
+
+        def handle_response(response):
+            try:
+                body = response.text()
+                match = jwt_pattern.search(body)
+                if match:
+                    token = match.group()
+                    found_tokens.append((f"network:{response.url}", token))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)